test/unit/test_disk_tensor.py

```diff
@@ -49,7 +49,7 @@ def test_readinto_read_speed(self):
       with Timing("copy in ", lambda et_ns: f" {test_size/et_ns:.2f} GB/s"):
         f.readinto(tst)
   def test_bitcasts_on_disk(self):
-    _, tmp = tempfile.mkstemp()
```

Comment:
changed to `temp('bitcast')` because `tempfile.mkstemp` returns an extra file handle which needs to be closed